client/galaxy/scripts/app/singleton.js

```diff
@@ -6,8 +6,10 @@ import addLogging from "utils/add-logging";
 import { GalaxyApp } from "./galaxy";
 import { serverPath } from "utils/serverPath";
 
-export function setGalaxyInstance(factory) {
-    console.warn("setGalaxyInstance", serverPath());
+export function setGalaxyInstance(factory, testing = false) {
```

Comment:
Maybe use the client side config system to detect whether testing or not?


```javascript
import config from "config";

export function setGalaxyInstance(factory) {
   let testing = config.someTestingFlagYouMake;
   ...
}

// client/galaxy/config/testing.js
export {
    name: "unit testing configs",
    debug: false,
    yourTestingFlag: true
    ...
}
```

Reply:
@Nerdinacan Feels bad to admit I forgot we added that; good idea.

Reply:
I probably would have forgotten too except I'm staring right at it for the rxdb configs.